docker/nginx/snippets/docs-redirects.conf

```diff
@@ -0,0 +1,4 @@
+# Document "Piano Triennale" - move permanently version "stabile" to version "2017-2019"
+# This is to prevent broken links pointing to version "stabile"
+# The version "stabile" needs to be deactivated
+rewrite ^/italia/piano-triennale-ict/pianotriennale-ict-doc/it/stabile/(.*)$ /italia/piano-triennale-ict/pianotriennale-ict-doc/it/2017-2019/$1 permanent;
```

Comment:
Questo codice è nuovo o è già presente altrove al momento?

Reply:
questa è la configurazione nginx attualmente in produzione